backend/app.py

```
@@ -0,0 +1,49 @@
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config)
+    CORS(app)
+    JWTManager(app)
+    cache.init_app(app)
+
+    limiter = Limiter(app=app, key_func=get_remote_address,
```

Ruff (0.11.9)

Local variable `limiter` is assigned to but never used

Remove assignment to unused variable `limiter`

(F841)